### PR TITLE
hotfix of the BUG number 1

### PR DESCRIPTION
**Pull request by AmK-Dev1**
i have fxed boha's BUG check it out

### CODE REVIEW

**Review by A-M-A96 — COMMENTED** [review]
okay

File: Login.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>BOHA DARHA KBIRA KHELA PAGE VIDE JAJAJAJAJAJ</h1>
+  </body>
```
Comment (A-M-A96) [inline Login.html:10]: `hahaha`

Comment (A-M-A96) [reply Login.html:10]: h

**Review by A-M-A96 — COMMENTED** [review]
check ,y changes

File: Login.html (hunk 1 above)
Comment (AmK-Dev1) [reply Login.html:10]: okay

**Pull request merged**